components/bl-granim-component/README.md

```diff
@@ -0,0 +1,97 @@
+| scrollDebounceThreshold | *Number*                                                                                                                                                                                                                  |               |       | NO           | YES        | The scroll debounce threshold (in ms).                                                                |
+| transitionSpeed         | *Number*                                                                                                                                                                                                                  |               |       | NO           | YES        | Transition duration between each gradient (in ms).                                                    |
+| loop                    | *Checkbox*                                                                                                                                                                                                                | `true`        |       | NO           | YES        | The animation loop.                                                                                   |
+
+## Events
+
+| Name               | Triggers                                   | Context Blocks                                                                                |
+|--------------------|--------------------------------------------|-----------------------------------------------------------------------------------------------|
+| On Start           | Triggered when the animation start.        |                                                                                               |
+| On Gradient Change | Triggered when a gradient change and loop. | ColorDetails: `{activeState: String, colorsFrom: Array, colorsTo: Array, isLooping: Boolean}` |
+| On End             | Triggered when the animation end.          |                                                                                               |
+
+## Examples
+
+Below is a Codeless Example highlighting how to use the Granim component:
+
+<p align="center">
+  <img src="./example-images/granim-view.png" alt="granim example view" width="780"/>
+</p>
+
+<details>
+<summary>Try yourself with this colors</summary>
+
+```
+[
+  [{
+    "color": "#833ab4",
+    "pos": 0.2
+  },
+    {
+      "color": "#fd1d1d",
+      "pos": 0.8
+    },
+    {
+      "color": "#38ef7d",
+      "pos": 1
+    }], [{
+      "color": "#40e0d0",
+      "pos": 0
+    },
+    {
+      "color": "#ff8c00",
+      "pos": 0.2
+    },
+    {
+      "color": "#ff0080",
+      "pos": 0.75
+    }]
+]
+```
+</details>
+
+Or another example:
+
+<p align="center">
+  <img src="./example-images/granim-example2.png" alt="granim example2" width="780"/>
+</p>
+<p align="center">
+  <img src="./example-images/granim-view2.png" alt="granim example2 view" width="780"/>
+</p>
```

Comment:
Please, use uniq content for examples. For now we have 3 components with the same list on images.